api/APIv2.md

```diff
@@ -363,11 +363,11 @@ Response:
 
 The response MUST contain no body content.
 
-### logs
+### newlogs
```

Comment:
How about keep the old and adding the new for now, since controllers need to provide both APIs for some time?

Reply:
will do.